static/specification.html

```diff
@@ -93,34 +106,208 @@
+          performs verification via a `POST` HTTP request.
+        </p>
+        <aside class="example">
+          <pre class="JS">
+            /** @type {MonetizationEvent} event */
+            async function verifyReceipt(event) {
+              const body = JSON.stringify({
+                receipt: event.receipt
+              });
+
+              const response = await fetch('https://verifier.example/verify/', {
+                method: 'POST',
+                headers: {
+                  'Content-Type': 'application/json'
+                },
+                body);
+              });
+
+              if (response.ok) {
+                // The payment was verified
+              } else {
+                // The payment was not verified
+              }
+            }
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h3>
+          Monetizing media
+        </h3>
+        <p>
+          The following example shows how to monetize various type of media
+          using different [=payment end-point=].
+        </p>
+        <aside class="example">
+          <pre class="html">
+            &lt;video src="myvideo" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/video/pay"&gt;
+            &lt;/video&gt;
+
+            &lt;audio src="music.mp3" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/audio/pay"&gt;
+            &lt;/audio&gt;
+
+            &lt;picture srcset="cat.jpeg" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/images/pay"&gt;
+            &lt;/picture&gt;
+          </pre>
+        </aside>
+      </section>
+    </section>
+    <section id="model">
       <h2>
-        Terminology
+        Model
       </h2>
       <dl>
         <dt>
-          <dfn data-lt="provider">Web Monetization provider</dfn>:
+          <dfn data-lt="monetized">Monetization</dfn>:
+        </dt>
+        <dd>
+          A payment made by a user to a website, facilitated through a
+          [=monetization provider=].
+        </dd>
+        <dt>
+          <dfn data-lt="provider">Monetization provider</dfn>:
         </dt>
         <dd>
-          The party making payments on behalf of the user. A Web Monetization
-          provider uses the [[[SPSP]]] (<abbr title=
-          "Simple Payment Setup Protocol">SPSP</abbr>) on top of the
-          [[[Interledger]]] (<abbr title=
-          "Interledger Protocol">ILP</abbr>)/[[[STREAM]]] to provide a
-          high-level way to send money and data.
+          The party making payments on behalf of the user. A monetization
+          provider leverages the [[[Interledger]]] suite of protocols and
+          technologies (e.g., [[Open-Payments]] or [[STREAM]]) to provide a
+          high-level way to pay a [=monetization receiver=].
```

Comment:
Mentioning Open Payments and STREAM in the same breath may be confusing since OP is the application layer and STREAM is the transport layer. 

Maybe say
> A monetization provider leverages the Interledger Protocol suite of protocols and technologies (e.g., [Open-Payments] or [SPSP] , both based on [STREAM]) to provide a high-level way to pay a monetization receiver.

Reply:
Love it! Thank you.